### ISSUE TITLE
The ToC is not displayed

### ISSUE THREAD

**Issue by RoneoOrg** [manuth/MarkdownConverter#67]
Hi,

put simply, the ToC is not appearing, neither in the HTML nor the PDF.

I did tests with the following configurations:

```
Version: 1.53.2
Commit: 622cb03f7e070a9670c94bae1a45d78d7181fbd4
Date: 2021-02-12T00:32:13.710Z
Electron: 11.2.1
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Linux x64 4.19.0-14-amd64
```

```
Version: 1.54.0-insider
Commit: e590188f17162393f50feec19263398e6fe02d13
Date: 2021-02-26T20:32:11.854Z
Electron: 11.3.0
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Linux x64 4.19.0-14-amd64 snap
```

in both cases with `Markdown converter v3.1.2`

I used for the tests a really basic `test.md` file:

```
# Titre
## Titre 2
## Titre 3
```

I even explicitly set `markdownConverter.Parser.Toc.Enabled`  to `true` in `settings.json`..  

still nothing

The files are properly generated, both HTML and PDF, but no ToC  
The HTML source shows nothing related to a ToC, and the differents logs in the Output console of Vscode stay empty

What am I missing? Is there any way to provide more debug information?

**Comment by manuth** [comment]
Hello @RoneoOrg
Thanks a lot for using MarkdownConverter!
Sorry to hear about you having trouble using the ToC-feature.
I totally forgot to point out on how to use it in the `README`.

MarkdownConverter provides a setting called `markdownConverter.ParsermToc.Indicator` which can be set to a regular expression to replace with the toc.

The default is `\[\[\s*toc\s*\]\]`, which matches `[[toc]]` with any number of spaces in between (e.g. `[[    toc    ]]`, `[[ toc ]]` etc.).

So basically you should make your `test.md` look like this:

```md
# Table des matières
[[ toc ]]

# Titre 1
## Titre 2
## Titre 3
```

Please let me know whether this helped.

**Comment by RoneoOrg** [comment]
**It works** with `[[toc]]`, both in an HTML and a PDF file.

I guess it could help others to add this info to the Readme (I would love to provide a PR, but won't be able to do so before 2 weeks or more..)

Thanks a lot for this awesome extension!

**Comment by RoneoOrg** [comment]
I opened an issue here: https://github.com/manuth/MarkdownConverter/issues/68
